config-object-store/src/main/java/org/hypertrace/config/objectstore/IdentifiedObjectStore.java

```diff
@@ -114,6 +116,33 @@ public List<ContextualConfigObject<T>> getAllObjects(RequestContext context) {
                 Collectors.toUnmodifiableList(), this::orderFetchedObjects));
   }
 
+  List<ContextualConfigObject<T>> getMatchingObjectsWithFilter(
```

Comment:
Raising a new thread in case the other comment got missed - https://github.com/hypertrace/config-service/pull/287#discussion_r2045120709

Reply:
Fixed it